aspnetcore/performance/caching/response.md

```diff
@@ -1,92 +1,72 @@
 
-   The maximum duration (in seconds) the response should be cached. **Required** unless `NoStore` is `true`.
+  There is no corresponding HTTP header for this property. This property is an HTTP feature handled by the response caching middleware. To set the `VaryByQueryKeys` property, the response caching middleware must be enabled.
 
-**Location `ResponseCacheLocation`**
+The `ResponseCacheAttribute` is used to configure and create (via `IFilterFactory`) a `ResponseCacheFilter`. The `ResponseCacheFilter` performs the work of updating the appropriate HTTP headers and features of the response. The filter:
 
-   The location where the response may be cached. May be `Any`, `None`, or `Client`. Default is `Any`.
+* Removes any existing headers for `Vary`, `Cache-Control`, and `Pragma`. 
+* Writes out the appropriate headers based on the properties set in the `ResponseCacheAttribute`. 
+* Updates the response caching HTTP feature if `VaryByQueryKeys` is set.
 
-**NoStore `bool`**
+### `Vary` 
 
-   Determines whether the value should be stored or not, and overrides other property values. When `true`, `Duration` is ignored and `Location` is ignored for values other than `None`.
+This header is only written when the `VaryByHeader` property is set. It is set to the `Vary` property's value. The following sample uses the `VaryByHeader` property.
 
-**VaryByHeader `string`**
+[!code-csharp[Main](response/sample/Controllers/HomeController.cs?name=snippet_VaryByHeader&highlight=1)]
 
-   When set, a `vary` response header will be written with the response.
+You can view the response headers with your browsers network tools or the [Fidder tool](http://www.telerik.com/fiddler). The following image shows the Edge F12 output on the **Network** tab when the `About2` action method is called:
 
-**CacheProfileName `string`**
+![Edge F12 output on the **Network** tab when the `About2` action method is called](response/_static/vary.png)
 
-   When set, determines the name of the cache profile to use.
-
-**Order `int`**
-
-   The order of the filter (from `IOrderedFilter`).
-
-The `ResponseCacheAttribute` is used to configure and create (via `IFilterFactory`) a `ResponseCacheFilter`, which performs the work of writing the appropriate HTTP headers to the response. The filter will first remove any existing headers for `Vary`, `Cache-Control`, and `Pragma`, and then will write out the appropriate headers based on the properties set in the `ResponseCacheAttribute`.
-
-### The `Vary` Header
+### `NoStore` and `Location.None`
 
-This header is only written when the `VaryByHeader` property is set, in which case it is set to that property's value.
+`NoStore` overrides most of the other properties. When this property is set to `true`, the `Cache-Control` header will be set to "no-store". If `Location` is set to `None`:
 
-### `NoStore` and `Location.None`
+* `Cache-Control` is set to `"no-store, no-cache"`. 
```

Comment:
I don't think that's a run on sentence if you keep the original wording? Then again, you are the language expert. The formatting just looks a little inconsistent with the rest of the docs. Just a minor aesthetics comment.